integration-tests/gradle/src/test/java/io/quarkus/gradle/devmode/BasicJavaMainApplicationModuleDevModeTest.java

```diff
@@ -0,0 +1,75 @@
+package io.quarkus.gradle.devmode;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.RandomAccessFile;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+
+@DisabledOnOs(OS.WINDOWS)
+public class BasicJavaMainApplicationModuleDevModeTest extends QuarkusDevGradleTestBase {
+    private static final String ARGS_LOG_LINE_PREFIX = "basic-java-main-application-project: args";
+
+    @Override
+    protected String projectDirectoryName() {
+        return "basic-java-main-application-project";
+    }
+
+    @Override
+    protected String[] buildArguments() {
+        return new String[] { "clean", "quarkusDev", "-s", "-Dquarkus.args=param1=1 param2=2 param3=3" };
+    }
+
+    protected void testDevMode() throws Exception {
+        System.out.println("testDevMode()...");
+        File logOutput = new File(projectDir, "command-output.log");
+        await().atMost(30, TimeUnit.SECONDS).until(() -> {
+            try {
+                String logLine;
+                RandomAccessFile commandOutputLogFile = new RandomAccessFile(logOutput, "r");
+                while ((logLine = commandOutputLogFile.readLine()) != null) {
+                    if (logLine.startsWith(ARGS_LOG_LINE_PREFIX)) {
+                        commandOutputLogFile.close();
+                        return true;
+                    }
+                }
+                commandOutputLogFile.close();
+                return false;
+            } catch (Exception e) {
+                System.out.println(String.format("e: <message, %s>, <cause, %s>", e.getMessage(), e.getCause()));
```

Comment:
Could this be replace by proper logging?

Reply:
@jacobdotcosta any chance you could take care of this request? Thanks!

Reply:
Hi! Sure thing. Didn't realize you already requested it a couple of weeks ago.

Reply:
I've replaced the System out with logging but...

@gsmet  checking the other tests they have no logs.  Should I simply remove the log message? I thought it would be interesting to have the error cause but I have no problem with eliminating it so this test is similar to others!